src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,16 +1,31 @@
 package racingcar
 
-class Car(initialMoveCount: Int) {
-    var moveCount: Int = initialMoveCount
+class Car(
+    initialMoveCount: Int = 0,
+    val name: String = "Car",
+) {
+    companion object {
+        private const val MOVE_MINIMUM_VALUE = 4
+    }
+
+    init {
+        if (name.length > 5) throw IllegalArgumentException("자동차 이름이 너무 길어요")
+    }
+
+    private var moveCount: Int = initialMoveCount
```

Comment:
```suggestion
   var moveCount: Int = initialMoveCount
        private set
```

private set을 사용하면, 내부에서만 수정 가능할 수 있어요. 이러면, status를 삭제할 수 있지 않을까요~?